src/prisma/schema.prisma

```diff
@@ -2086,3 +2088,17 @@ model CharmTransaction {
   toWallet   CharmWallet? @relation("toWallet", fields: [to], references: [id], onDelete: SetNull)
   metadata   Json?        @default("{}")
 }
+
+model FavoriteCredential {
+  id                   String            @id @default(uuid()) @db.Uuid
+  index                Int               @default(-1)
+  chainId              Int?
+  userId               String            @db.Uuid
+  user                 User              @relation(fields: [userId], references: [id], onDelete: Cascade)
+  attestationId        String?
+  issuedCredentialId   String?           @db.Uuid
+  issuedCredential     IssuedCredential? @relation(fields: [issuedCredentialId], references: [id], onDelete: SetNull)
+  gitcoinWalletAddress String?
+
+  @@unique([issuedCredentialId, attestationId, gitcoinWalletAddress])
```

Comment:
@@unique[] defines a composite index with all 3 values

Instead here, you should @unique each individual field